### PR TITLE
Feature/P2022 662 movie API

### PR DESCRIPTION
**Pull request by MS151994**

### CODE REVIEW

File: pages/api/movie/[id].js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    title: "Fight Club",
+    video: false,
+    vote_average: 8.4,
+    vote_count: 23388,
+  },
+];
+
+export default async function handler(req, res) {
+  const { id } = req.query;
+  const findMovie = movies.find((movies) => movies.id === Number(id));
```
Comment (marcelina-hasiak) [inline pages/api/movie/[id].js:76]: Just a little proposal: `const findMovie = movies.find(movie => movie.id === Number(id));`

**Review by marcelina-hasiak — COMMENTED** [review]
Looks good ;)

File: pages/api/movie/[id].js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    title: "Fight Club",
+    video: false,
+    vote_average: 8.4,
+    vote_count: 23388,
+  },
+];
+
+export default async function handler(req, res) {
+  const { id } = req.query;
+  const findMovie = movies.find((movie) => movie.id === Number(id));
```
Comment (danielmajewski) [inline pages/api/movie/[id].js:76]: `findMovie` name suggests function name. Use more like: `movieFound` or `movieSearchResult`

File: pages/api/movie/[id].js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  findMovie ? res.status(200).json(findMovie) : res.status(200).json({});
```
Comment (danielmajewski) [inline pages/api/movie/[id].js:77]: It is not really readable and I would suggest:
```suggestion
  res.status(200).json(movieSearchResult || {});
```
or
```suggestion
  responseData = movieSearchResult || {};
  res.status(200).json(responseData);
```

File: pages/api/movie/[id].js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      { english_name: "English", iso_639_1: "en", name: "English" },
+    ],
+    status: "Released",
+    tagline: "Mischief. Mayhem. Soap.",
+    title: "Fight Club",
+    video: false,
+    vote_average: 8.4,
+    vote_count: 23388,
+  },
+];
```
Comment (danielmajewski) [inline pages/api/movie/[id].js:72]: can you move it to another file called `movieMockData` in json file?

**Review by danielmajewski — APPROVED** [review]

**Comment by wlechowicz** [comment]
There's a little bit of a mess right now since you copied this and added it as part of PR #30 so I think this PR should be closed and this code will get merged in #30

**Pull request closed without merging**